mtags/src/main/scala-2/scala/meta/internal/pc/MetalsGlobal.scala

```diff
@@ -324,8 +318,11 @@ class MetalsGlobal(
             history.config.get(ownerSymbol) match {
               case Some(rename) if canRename(rename, ownerSymbol) =>
                 TypeRef(
-                  new PrettyType(rename.toString),
-                  backtickifiedSymbol,
+                  SingleType(
```

Comment:
Replacing PrettyType with SingleType wa enough except for CompletionWorkspaceSuite.type-import where the type would alwyas be dealiased when printing. So I changed the symbol to always be error in shortened type.